iceoryx2-cal/src/communication_channel/message_queue.rs

```diff
@@ -281,9 +289,7 @@ impl<T: Copy + Debug> CommunicationChannelCreator<T, Channel<T>> for Creator<T>
         };
 
         // we are finished with the setup and we open the channel for others to connect
-        _shared_memory
-            .set_permission(Permission::OWNER_READ | Permission::OWNER_WRITE)
-            .unwrap();
+        _shared_memory.set_permission(FINAL_PERMISSIONS).unwrap();
```

Comment:
This is a semantic change. Is the exec flag necessary?

Reply:
Nope, not really but `Permission::OWNER_READ | Permission::OWNER_WRITE` is not const and my thought was that the `exec` flag does not cause any harm here so I added it.